osquery/tables/system/windows/logon_sessions.cpp

```diff
@@ -0,0 +1,93 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+// clang-format is turned off to ensure windows.h is first
+// clang-format off
+#include <Windows.h>
+// clang-format on
+#include <NTSecAPI.h>
+#include <sddl.h>
+#include <tchar.h>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/process_ops.h"
+#include "osquery/filesystem/fileops.h"
+
+#define STATUS_SUCCESS ((NTSTATUS)0x00000000L)
```

Comment:
Just a question, why do you need to redefine `STATUS_SUCCESS` here? Do we have a chance to use it directly from `Windows.h`?

Reply:
I tried to use it the value from `Windows.h`, but intellisense looked like it wasn't happy with it. That being said, i never tried to compile to see if it'd just work.

Reply:
<img width="1057" alt="screen shot 2018-07-26 at 7 02 24 am" src="https://user-images.githubusercontent.com/13278210/43267109-e9ae1252-90a1-11e8-896a-106276bdce04.png">
This is the error i get when i remove the `STATUS_SUCCESS` definition. I'll admit that I'm new to C++ and could be doing something naive.

Reply:
I am not a jedi in windows API, but maybe you need additional include of [winerror.h](https://docs.microsoft.com/en-us/windows/desktop/api/winerror/) ?

Reply:
Looks like `ntstatus.h` worked 👍

Reply:
Actually, it appears to work, but then when i compile i get some weird warning that is interpreted as an error. I honestly have no idea how to fix this because it doesn't appear that it likes me including `winnt.h` instead of `ntstatus.h`.

```
c:\program files (x86)\windows kits\10\include\10.0.17134.0\shared\ntstatus.h(1365): warning C4005: 'DBG_COMMAND_EXCEPTION': macro redefinition 
c:\program files (x86)\windows kits\10\include\10.0.17134.0\um\winnt.h(2463): note: see previous definition of 'DBG_COMMAND_EXCEPTION'
```